docs/general/rendering.md

```diff
@@ -73,7 +73,11 @@ See [ARCHITECTURE](../ARCHITECTURE.md) for more information.
 
 #### fields
 
-PR please :)
+```ruby
+render json: @user, fields: [:access_token]
```

Comment:
This is only valid for the `json` and `attributes` adapter. For the `json_api` adapter, you would use 

```ruby
render json: @user, fields: { users: [:access_token] }
```

Where `users` is the JSONAPI type.

Could you add the missing code snippet with a brief description (stating for which adapter each snippet is valid) or add a description above your code that states that it only valid for `json` and `attributes` adapter?